docker-app/qfieldcloud/core/tests/test_delta.py

```diff
@@ -282,7 +282,7 @@ def test_push_apply_delta_file_not_json(self):
 
         response = self.client.post(
             f"/api/v1/deltas/{project.id}/",
-            {"file": open(delta_file, "r")},
+            {"file": open(delta_file)},
```

Comment:
Any chance we disable this rule. To me it is much more readable with explicit `open` flags, rather than memorizing the `open` defaults.

Reply:
I haven't found any options to do this (there are options for minimum python version `--py37-plus` etc, `--keep-percent-format`, `--keep-mock`, `--keep-runtime-typing`, none of the later ones looks particularly interesting to me) ...

I guess we either

- apply the changes and remove the pre-commit
- use all the rules and try to get used to this

Reply:
Let's go to with the latter then.